src/server/puppet-implementation.ts

```diff
@@ -1375,6 +1375,8 @@ function puppetImplementation (
         response.setName(payload.name)
         response.setRoomAlias(payload.roomAlias || '')
         response.setAdditionalInfo(payload.additionalInfo || '')
+        response.setJoinScene(payload.joinScene || PUPPET.types.RoomMemberJoinScene.Unknown)
+        response.setJoinTime(payload.joinTime || Date.now())
```

Comment:
I think we should not set it to Date.now() in puppet-service level. This properties is optional so if there is no such value, just leave it blank.